OSSSpeechKit/Classes/OSSSpeechUtility.swift

```diff
@@ -72,33 +73,39 @@ public class OSSSpeechUtility: NSObject {
     
 }
 
-extension NSObject {
-    /// Method outputs a debug statement containing necessary information to resolve issues.
-    ///
-    /// Only works with debug/dev builds.
-    ///
-    ///  - Parameters:
-    ///     - object: Any object type
-    ///     - functionName: Automatically populated by the application
-    ///     - fileName: Automatically populated by the application
-    ///     - lineNumber: Automatically populated by the application
-    ///     - message: The message you wish to output.
-    public func debugLog(object: Any, functionName: String = #function, fileName: String = #file, lineNumber: Int = #line, message: String) {
-        #if DEBUG
-        let className = (fileName as NSString).lastPathComponent
-        print("\n\n******************\tBegin Debug Log\t******************\n\n\tClass: <\(className)>\n\tFunction: \(functionName)\n\tLine: #\(lineNumber)\n\tObject: \(object)\n\tLog Message: \(message)\n\n******************\tEnd Debug Log\t******************\n\n")
-        #endif
-    }
-}
+#endif
 
 /// Bundle extension to aid in retrieving the SDK resources for getting SDK images.
 extension Bundle {
-    /// Will return the Bundle for the SDK if it can be found.
-    static func getResourcesBundle() -> Bundle? {
-        let bundle = Bundle(for: OSSSpeech.self)
-        guard let resourcesBundleUrl = bundle.resourceURL?.appendingPathComponent("OSSSpeechKit.bundle") else {
-            return nil
-        }
-        return Bundle(url: resourcesBundleUrl)
-    }
+	/// Will return the Bundle for the SDK if it can be found.
+	static func getResourcesBundle() -> Bundle? {
```

Comment:
Unless I am not mistaken, indentation here has increased, which is probably due to the `#if` statement, Xcode always messes up indentation when implementing those macros.